packages/docusaurus-theme-live-codeblock/src/theme/CodeBlock/index.js

```diff
@@ -60,7 +74,7 @@ export default ({
       <Playground
         scope={{...React}}
         code={children.trim()}
-        theme={prism.theme || defaultTheme}
```

Comment:
By the way prism.theme is no longer working now

Reply:
I replaced this line with "theme={prismTheme}", so it should work, does not it?

Reply:
It doesn't. Look at this PR, no more mention of `prism.theme`. Light theme is always defaultTheme (which is palenight). 

Just try adding prism.theme on docusaurus own config.

Reply:
oh my gash, now I understand what you meant! Thanks for pointing this out! Fixed :heavy_check_mark: 
I also moved the hook to theme.